src/main/java/com/uci/orchestrator/Consumer/ReactiveConsumer.java

```diff
@@ -22,10 +22,7 @@
 import io.fusionauth.domain.UserRegistration;
 import lombok.RequiredArgsConstructor;
 import lombok.extern.slf4j.Slf4j;
-import messagerosa.core.model.DeviceType;
-import messagerosa.core.model.SenderReceiverInfo;
-import messagerosa.core.model.Transformer;
-import messagerosa.core.model.XMessage;
+import messagerosa.core.model.*;
```

Comment:
It's better to import specific classes instead of whole package.

Reply:
Yes, you are right but when we use more than 5 classes from same package IDE automatically do this. I will try to figure out next time.